internal/exporter/pollingexporter/README.md

```diff
@@ -0,0 +1,28 @@
+# Polling Exporter
```

Comment:
Would something like "http testing" or "sanity" exporter be more accurate and denote this is for testing purposes only?

Reply:
Yes polling isn't really a good name. I don't want to call it testing as that implies automated testing and this can be useful for manual tests as well but for sure that'd be better than polling )

Reply:
Renamed to `httpsink`.